_py-rse.yml

```diff
@@ -7,20 +7,21 @@ language:
     chapter_name: "Chapter "
 output_dir: "_book/py-rse"
 delete_merged_file: false
+before_chapter_script: "_common.R"
 rmd_files:
   - index.Rmd
   - py-rse/bash-basics.Rmd
   - py-rse/bash-advanced.Rmd
   - py-rse/scripting.Rmd
+  - py-rse/style.Rmd
   - py-rse/git-cmdline.Rmd
   - py-rse/git-advanced.Rmd
+  - py-rse/teams.Rmd
   - py-rse/automate.Rmd
   - py-rse/config.Rmd
   - py-rse/errors.Rmd
-  - py-rse/teams.Rmd
-  - py-rse/style.Rmd
-  - py-rse/project.Rmd
   - py-rse/testing.Rmd
```

Comment:
```suggestion
  - py-rse/testing.Rmd
  - py-rse/publish.Rmd
```

Reply:
I've put `publish.Rmd` in the place where `provenance.Rmd` will go (and #422 will change the name of the file and update the contents).